packages/react-core/src/components/NotificationDrawer/NotificationDrawerListItem.tsx

```diff
@@ -52,6 +63,7 @@ export const NotificationDrawerListItem: React.FunctionComponent<NotificationDra
       onClick={(e) => onClick(e)}
       onKeyDown={onKeyDown}
     >
+      <span className="pf-v5-screen-reader">{readStateSRText}</span>
```

Comment:
could remove 44-52 and do this

```suggestion
      <span className="pf-v5-screen-reader">{readStateScreenReaderText || (isRead ? 'read' : 'unread')}</span>
```